inductiva/templates/md_water_box/gromacs/commands.json.jinja

```diff
@@ -1,9 +1,9 @@
 [
     {"cmd": "gmx solvate -cs tip4p -box {{ box_size }} -o conf.gro -p topol.top", "prompts": []},
     {"cmd": "gmx grompp -f energy_minimization.mdp -o min.tpr -pp min.top -po min.mdp -c conf.gro -p topol.top", "prompts": []},
-    {"cmd": "gmx mdrun -s min.tpr -o min.trr -x min.xtc -c min.gro -e min.edr -g min.log", "prompts": []},
+    {"cmd": "gmx mdrun -s min.tpr -o min.trr -c min.gro -e min.edr -g min.log", "prompts": []},
```

Comment:
the -x min.xtc is the flag for creating a trajectory file and it is not needed for this step, we just want the last one, the eql.xtc.

Reply:
The last one, which is the compressed?

Reply:
I know I am being picky here, but what thus the compress mean here? Is it a selection of "frames" of the full trajectory? Is the frame rate already adequate for most vizs?

Reply:
GROMACS generates the precision trajectory .trr file which contains the forces, the velocities and the coordinates of every atom. The compressed .xtc file contains only the coordinates taken every x steps. This is why the trr file has GB of size and the compressed file has only some MB. In this case I defined it as being every 50 steps. Which given the timestep of 0.2 fentoseconds is every 10 fs. The framerate is defined elsewhere, when you render the video.

Reply:
Wonderful thank you for the explanation! With it, I notice that the 'output_time_step' is not important for the rendering, since the simulations cannot be visualized in real time :D
For processing the video the frame rate can also be one parameter for the user.

Reply:
That's a good idea :) I will make that configurable in the jinja file.

Reply:
But it needs to be in the commands?

Reply:
Not in the commands, it is defined in .mdp files - the parameters files!